src/Command/InfectionCommand.php

```diff
@@ -355,14 +356,15 @@ private function runMutationTesting(TestFrameworkAdapter $adapter): void
 
         $processBuilder = new MutantProcessBuilder($adapter, $this->versionParser, $config->getProcessTimeout());
 
-        $codeCoverageData = $this->getCodeCoverageData($this->testFrameworkKey, $adapter);
```

Comment:
[This method definition](https://github.com/infection/infection/blob/131d75c9e71c3aa690c05c8452ece5c19c22330e/src/Command/InfectionCommand.php#L491-L504) should be removed since it's unsed after this change

Reply:
👍